.github/scripts/updateChangelog.js

```diff
@@ -6,12 +6,19 @@
  * @param {string} releaseDate the release date to be written to the changelog
  */
 function updateChangeLog(changelogString, version, releaseDate) {
-    const optionalBetaOrRc = '(-rc.d+)?(-(b|beta).d+)?';
-    const changelogVersionRegExp = new RegExp(
-      `\\[(development|develop|unreleased|${version})${optionalBetaOrRc}.*`,
-      'gi',
-    );
-    return changelogString.replace(changelogVersionRegExp, `[${version}] - ${releaseDate}`);
-  }
-  
-  module.exports.updateChangeLog = updateChangeLog;
+  const optionalBetaOrRc = '(-rc.d+)?(-(b|beta).d+)?';
+  const changelogVersionRegExp = new RegExp(
+    `\\[(development|develop|unreleased|${version})${optionalBetaOrRc}.*`,
+    'gi',
+  );
+
+  const lastReleaseVersion = changelogString.match(/\[(\d+.\d+.\d+)\]/)[1];
+  changelogString = changelogString.replace(changelogVersionRegExp, `[${version}] - ${releaseDate}`);
+
+  return changelogString.replace(
+    '## 1.0.0 (2017-02-03)\n- First release\n\n',
+    `## 1.0.0 (2017-02-03)\n- First release\n\n[${version}]: https://github.com/bitmovin/bitmovin-player-ui/compare/v${lastReleaseVersion}...v${version}\n`
```

Comment:
interesting approach 😅 

Alternatively, we could have added a comment above the compare link section, and matched that 🤔 
Or matched the previous comparison link.

but I guess this is fine too, as it shouldn't change.

Reply:
I tried that before, but this is less complicated and has less chances of bug😄